tooling/api/src/window.ts

```diff
@@ -45,6 +45,16 @@ export interface Monitor {
 type Theme = 'light' | 'dark'
 type TitleBarStyle = 'visible' | 'transparent' | 'overlay'
 
+type ResizeDirection =
+  | 'East'
```

Comment:
should it be camelCase?

Reply:
I think the current casing makes sense for directions even though it'll be different from the other defined types. Not sure i'd like to rename the other types though so maybe a mix is fine?
Or maybe we should add an alias on the rust side so non ts users won't get a stroke?